src/builder/modules/Generator.res

```diff
@@ -130,6 +130,10 @@ let fillBackgroundColorWithWhite = () => {
   model := Generator_Builder.fillBackgroundColor(model.contents, "#ffffff")
 }
 
+let fillRect = (dest: Builder.rectangle, color: string) => {
```

Comment:
Hi @NinjolasNJM for each of these functions, let's justify their purpose in the context of papercraft designs.

For `fillRect()` I can see this might be useful as a background fill behind things that are white (so we can see the edges), perhaps? Any other uses you have in mind?

Reply:
the fillRect() function I made so that the mini generator could have the faces of the head that need to be glued on highlighted the same way they were in the original design- similar uses could be to eliminate the need for images on other colored labels.

Reply:
> so that the mini generator could have the faces of the head that need to be glued on highlighted the same way they were in the original design

Are you referring to this part?

![image](https://user-images.githubusercontent.com/85809598/171986991-9e11b193-e5f4-491a-8aa6-d893ee026b66.png)

Could you explain the idea a bit more (I forgot what did on the original design)?

> images on other colored labels.

That's a great example. Actually we could probably benefit from a `drawLabel()` function with an optional background colour eventiually.

Reply:
Yes- in the original design, those faces were like this:
![1328909756651](https://user-images.githubusercontent.com/16890123/172025250-6b4805d0-643c-4b53-a113-81de917ee43a.png)
In this PR, I added this change to the mini generator, along with having it use the new line and fold functions.

>  Actually we could probably benefit from a drawLabel() function with an optional background colour eventiually.

This is a good idea for a function that could work well with fleshing out the text drawing function- after this PR, its use could be expanded, and maybe a sign generator could be made using it.